requirements.txt

```diff
@@ -100,6 +100,7 @@ Twisted==17.5.0
 twitter==1.17.1
 Unidecode==0.4.21
 urllib3==1.22
```

Comment:
Should we double check all the packages to remove any useless one?

Reply:
yes, good idea. will do that!

Reply:
Just checked, the `twitter` package is the only one that is not required......